### PR TITLE
Typings

### LINKED ISSUE #78: TS Typings for Editor Support
I'm happy to start a PR that adds `.d.ts` files for editor plugin (i.e. typescript language-server) support.

### PR DESCRIPTION
**Pull request by bennypowers**
This PR adds typescript typings and closes #78 

Please pay particular attention to `packages/storybook/index.d.ts`

This PR also incidentally adds `.js` to some relative imports in `semantic-dom-diff`, in anticipation of some future where bare and url specifiers can coexist. I though it was a minor enough change to include, if deemed out of scope i'll rebase.

### CODE REVIEW

**Comment by daKmoR** [comment]
oups - sorry I pressed the "Update branch" button and it merged the master in... while I was expecting a rebase 🙈

File: packages/testing-helpers/lit-html.d.ts [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export { render, html } from 'lit-html/lit-html';
+export function unsafeStatic<TOptions>(options: TOptions): { d: TOptios }
```
Comment (daKmoR) [inline packages/testing-helpers/lit-html.d.ts]: TOptios

File: packages/storybook/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+export { html } from 'lit-html';
+
+// NB: @types/storybook__polymer doesn't yet exist
+// export { storiesOf, addParameters } from '@storybook/polymer';
```
Comment (daKmoR) [inline packages/storybook/index.d.ts]: anything we can do about this? the "types" would be the same as for `@storybook/foo`

File: packages/testing-helpers/lit-html.d.ts (hunk 1 above)
Comment (daKmoR) [reply packages/testing-helpers/lit-html.d.ts]: changed it 🤗

**Comment by bennypowers** [comment]
I'm happy to merge now and deal with things later.

**Comment by daKmoR** [comment]
agreed 👍 - merged 🎉 

ci will release it in a couple of minutes 🤗

**Pull request merged**